### PR TITLE
[Feature 1] Build initial screen layout

### PR DESCRIPTION
**Pull request by alexxispn**
## Description

This pull request adds three new components to the project: a AppLinkButton, a LogoFesbalIcon, and a WelcomeScreen.

## Changes

- Add AppLinkButton component. It's a Link with button style that can be used to navigate to a different page.
- Add LogoFesbalIcon component. It's exported from Figma design and converted to a component, it's the company logo.
- Add WelcomeScreeen component. It contains the logo and 2 AppLinkButton that should lead the user to the registration page or to the login page(registration page is not implemented yet)
- Update ButtonProps and little refactor AppNextButton.

## Checks

- [X] Project Builds
- [X] Project passes tests and checks
- [x] Updated documentation accordingly
- [X] Both buttons are enabled. The “Crear cuenta” button leads to the following issue (issue 1.2)
- [X] Screen follows provided designs

## Additional information

Reference: [1] Build initial screen layout
URL: https://github.com/TheTributeCommunity/fesbal-frontend/issues/2

### CODE REVIEW

File: src/types/ButtonProps.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 interface ButtonProps {
-    disabled: boolean;
     title: string;
+    disabled?: boolean;
+    bgColor?: string;
+    toGo?: string;
```
Comment (BorjaZarco) [inline src/types/ButtonProps.ts:5]: I would say that the `toGo` parameter does not belong to any Button, just the one that includes navigation. For that reason, I would create a new type (perhaps one that extends from the `ButtonProps`) exclusively for the navigation button. 

Check out my suggestion on the `AppLinkButton`

File: src/components/atom/AppLinkButton.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import ButtonProps from "../../types/ButtonProps";
+import {FC} from "react";
+import {Link} from "react-router-dom";
+
```
Comment (BorjaZarco) [inline src/components/atom/AppLinkButton.tsx:4]:
```suggestion
interface AppLinkButtonProps extends ButtonProps {
    toGo: string
}
```

I would create a new type only for this special kind of button where it is expected to always receive a `toGo` string

**Review by BorjaZarco — APPROVED** [review]
Left a couple of suggestions, the relevant are the ones regarding the AppLinkButton props. Apart from that LGTM! 👍 

I will leave my approval, but I would highly recommend checking out the relevant suggestions before merging

File: src/components/atom/AppNextButton.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 import ButtonProps from "../../types/ButtonProps";
 
-const AppNextButton: FC<ButtonProps> = ({title, disabled}) => {
+const AppNextButton: FC<ButtonProps> = ({title, disabled = false, bgColor = "bg-primary-color"}) => {
     return (
         <button
-            className="rounded-2xl flex items-center justify-center py-5 text-white font-bold bg-primary-color
-    disabled:bg-white disabled:text-primary-color border border-primary-color disabled:opacity-50"
+            className={`rounded-2xl flex items-center justify-center py-5 text-white font-bold ${bgColor}
+            ${disabled ? "disabled:bg-white disabled:text-primary-color border border-primary-color disabled:opacity-50" : ""}`}
```
[the remark this answers is not in the document]
Comment (alexxispn) [reply src/components/atom/AppNextButton.tsx:8]: Hi @BorjaZarco 

I have made all changes that you have recommend and implement them on the new PR I just create: [Issue2](https://github.com/TheTributeCommunity/fesbal-frontend/pull/52).

Thank you for your feedback and let me know if there is something that I should take into account.

File: src/pages/WelcomeScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import {useTranslation} from "react-i18next";
+import {namespaces} from "../i18n/i18n.constants";
+import LogoFesbalIcon from "../components/icons/LogoFesbalIcon";
+import AppLinkButton from "../components/atom/AppLinkButton";
+
+const WelcomeScreen = () => {
+    const {t} = useTranslation(namespaces.pages.welcomeScreen);
```
Comment (yodra) [inline src/pages/WelcomeScreen.tsx:7]: I commented on it in another PR, but it's better if we use legible names, for example, `translate` so when you use this hook inside the component you have more context to understand what `t` does 🙂

**Review by yodra — COMMENTED** [review]
Please @alexxispn check the comments pending to resolve to approve this PR 😉

File: src/pages/WelcomeScreen.tsx (hunk 4 above)
Comment (alexxispn) [reply src/pages/WelcomeScreen.tsx]: Thanks for bringing that up. I have pushed a new commit with updated naming to make it more legible. Let me know if there's anything else you'd like me to change slightly_smiling_face

**Review by yodra — APPROVED** [review]

**Pull request merged**